### DISCUSSION TITLE
Results

### DISCUSSION THREAD

**Discussion by pabloatria18** [theislab/cellrank#611] [Q&A] [answered]
Hello everyone!
I am fairly new to the scRNA-seq analysis and RNA velocity.
I've been trying to analyze some data (I currently have 2 conditions) and I need help / guidance interpreting and selecting the analysis I want to use.
First of all, I ran the scVelo part previous to the CellRank, and I kept obtaining results that do not make sense

For example, pink cluster is the most differentiated one, and green is an intermediate state, so it does not make sense the vector's direction
<img width="253" alt="Subset3" src="https://user-images.githubusercontent.com/54920011/121735465-29494300-cac4-11eb-8aba-ae94dd836a56.png">

When I run the dataset with this guideline: https://cellrank.readthedocs.io/en/stable/beyond_rna_velocity.html
It makes more sense:
<img width="799" alt="Screen Shot 2021-06-11 at 2 59 36 PM" src="https://user-images.githubusercontent.com/54920011/121736525-ad4ffa80-cac5-11eb-8976-76e1216d07e6.png">

I appreciate your help / opinions

**Comment by pabloatria18** [comment]
@Marius1311 I just noticed in another discussion that you mentioned that you don't get notifications for new discussions. So I am just tagging you in case no one sees this! Thanks!

**Comment by Marius1311** [comment, marked as answer]
Hi @pabloatria18! This looks interesting! I did in fact not get a notification for this  - @michalk8, can you look into the settings please and change them so that we get notified when new questions are opened? Thanks!

RNA velocity does in fact not seem to work very well on your data - the arrows point from most differentiated towards less differentiated. To figure out what the problem with the actual velocity vectors is, I would recommend opening a question in [scVelo](https://github.com/theislab/scvelo/discussions) as that's not really related to CellRank per se. 

As to the second part of your analysis, please be aware of the fact that you're not using RNA velocity here - you're using CytoTRACE to infuse directionality into the KNN graph. I have never worked with CytoTRACE in a perturbation setting, so it's important that you read the [original publication](https://science.sciencemag.org/content/367/6476/405) to get familiar with the biological assumptions made to check whether these hold for your system. 

Assuming for now that these assumptions are met, your trajectory analysis indicates that upon your perturbation ('experimental'), cells acquire a higher probability of transitioning towards cluster 5. You can quantify this fate bias by computing fate probabilities towards clusters 5 & 6, see e.g. the [kernels and estimators](https://cellrank.readthedocs.io/en/stable/kernels_and_estimators.html) tutorial.

**Reply by pabloatria18** [reply]
Thank you very much @Marius1311 !
I will do what you suggest!

Best !
